bots/machine/machine_core/cli.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
This leaves empty line on the top of the file. I'll fix it on all files that are affected